ui/desktop/electron-app/src/app-updater.js

```diff
@@ -4,11 +4,12 @@ const https = require('https');
 const semver = require('semver');
 const { parse } = require('node-html-parser');
 const { autoUpdater, dialog } = require('electron');
-const { releaseVersion } = require('../config/forge.config.js');
+// TODO: Pickup app release version from config file
+// const { releaseVersion } = require('../config/forge.config.js');
 
 const debug = process.env.DEBUG_APP_UPDATER;
 const releasesUrl = 'https://releases.hashicorp.com/boundary-desktop/';
-let currentVersion = releaseVersion;
+let currentVersion = '1.0.0';
```

Comment:
Setting it here for release as picking it up as part of config isn't working. Will continue to work on this.

Reply:
Let's plan to automate this for next release.